tools/darwin/Configurations/App-OSX.xcconfig

```diff
@@ -24,7 +24,7 @@ ARCHS = i386 x86_64
 ONLY_ACTIVE_ARCH = YES
 
 SDKROOT = macosx
-MACOSX_DEPLOYMENT_TARGET = 10.7
+MACOSX_DEPLOYMENT_TARGET = 10.8
```

Comment:
needs a bump here too

https://github.com/xbmc/xbmc/blob/master/tools/depends/configure.ac#L257

not sure if anywhere else for cmake